### PR TITLE
add a python script to show the runtime packages' dependencies

### PR DESCRIPTION
**Pull request by guanqun**
And see how messy it is  :)

### CODE REVIEW

File: scripts/runtime-dep.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# To run this script, you need to install the 'toml' python package and install the 'graphviz' package:
```
Comment (bkchr) [inline scripts/runtime-dep.py]: You should add `#!/usr/bin/env python3` to the top and make the script executable. 
Then you don't need to call it with `python ./scripts/runtime-dep.py`.

**Comment by ordian** [comment]
FYI, there is https://github.com/kbknapp/cargo-graph.

File: scripts/runtime-dep.py (hunk 1 above)
Comment (guanqun) [reply scripts/runtime-dep.py]: I want to user to specify their own version of python. ;)  Kidding, fixed now.

**Comment by guanqun** [comment]
@ordian The last time I looked at this tool, it doesn't have the option to specify a list of packages, it just list all the depedencies and draw the graph, but here, I only care about the modules in runtime.

**Comment by guanqun** [comment]
oops, just noticed that the package name is modified... Let me update it.

**Pull request merged**